oracle/src/lib.rs

```diff
@@ -121,6 +121,14 @@ pub mod module {
 	pub type IsUpdated<T: Config<I>, I: 'static = ()> =
 		StorageMap<_, Twox64Concat, <T as Config<I>>::OracleKey, bool, ValueQuery>;
 
+	/// Time until which the combined value with the given key is valid. Note
+	/// that the value if wrapped in an Option because MomentOf does not
+	/// implement Default
+	#[pallet::storage]
+	#[pallet::getter(fn expiration_time)]
+	pub type ExpirationTime<T: Config<I>, I: 'static = ()> =
+		StorageMap<_, Twox64Concat, <T as Config<I>>::OracleKey, Option<MomentOf<T, I>>, ValueQuery>;
```

Comment:
```suggestion
		StorageMap<_, Twox64Concat, <T as Config<I>>::OracleKey, MomentOf<T, I>, OptionQuery>;
```

Reply:
Yep, looks like the `Moment` associated type should support `Default`: https://github.com/paritytech/substrate/blob/master/frame/support/src/traits/misc.rs#L169